hack/kustomize/test-integration.sh

```diff
@@ -68,6 +68,7 @@ jq -c '.Extra[]' "$RES" | sort > extra.jsonl
 
 if [[ $DIFF_COUNT -gt 0 ]] ; then
   red "MISSING: $(jq "(.Missing | length)" "$RES")"
+  cat missing.jsonl
```

Comment:
You intended to add this beyond debugging while working?

Reply:
Yes, i thought it would be useful in the future to know what metrics were missing in a CI run